homeassistant/components/sensor/filter.py

```diff
@@ -446,7 +445,8 @@ class TimeSMAFilter(Filter):
         variant (enum): type of argorithm used to connect discrete values
     """
 
-    def __init__(self, window_size, precision, entity, type):
+    def __init__(self, window_size, precision, entity,
+                 type):  # pylint: disable=redefined-builtin
```

Comment:
That looks weird too.

Reply:
I think it's the least intrusive way to achieve the desired effect. Keeping it all on same line would require additional `noqa` comment for flake8.